lib/validate.py

```diff
@@ -31,4 +36,119 @@ def errorMessage(value_number, expected_range, actual_value, fileName):
 
     """
     print "[Unexpected Value] of Arg", value_number, " @", fileName, "| EXPECTED_RANGE:", \
-        expected_range, "| GOT:", actual_value
\ No newline at end of file
+        expected_range, "| GOT:", actual_value
+        
+        
+# Correlational : statistical distribution as illustrated by box plot for degree
+def box_plot_for_degree(log_directory, output_directory, channel_name):
```

Comment:
It's better to make a list parameter for channel names and iterste over the list. Right now, the list is static.